### ISSUE TITLE
Procuring image metrics

### ISSUE THREAD

**Issue by bdeck8317** [poldracklab/mriqc#882]
Hi there,

Not technically an issue. I was wondering if there was a way to obtain the image metrics which are shared in your data sharing agreement. I would like to compare the metrics derived from my images after running MRI_qc to those stored in your data sharing repository. 

Is this possible?

Have I simply missed the documentation on this?

Thanks!

**Comment by oesteban** [comment]
> Not technically an issue. I was wondering if there was a way to obtain the image metrics which are shared in your data sharing agreement. I would like to compare the metrics derived from my images after running MRI_qc to those stored in your data sharing repository.
> 
> Is this possible?

If I correctly understood what you want to do, yes, this is actually the idea for our crowdsourced database. This is described with some depth in our paper (https://doi.org/10.1038/s41597-019-0035-4).

> Have I simply missed the documentation on this?

Probably not, the documentation is pretty much scattered and hard to follow. Luckily everything is out there to piece together.

For instance, an example of how to pull IQMs from the server - https://github.com/poldracklab/mriqc/blob/master/docs/notebooks/MRIQC%20Web%20API.ipynb

**Comment by bdeck8317** [comment]
@oesteban  awesome this is very helpful!
